function/config/default.json

```diff
@@ -1,5 +1,6 @@
 {
   "samUrl": "https://sam.dsde-dev.broadinstitute.org/",
   "rawlsUrl": "https://rawls.dsde-dev.broadinstitute.org/",
-  "whitelistUrl": "gs://davidan-sts-test-whitelist-1/whitelist.json"
+  "whitelistBucket": "davidan-sts-test-whitelist-1",
```

Comment:
Will testers/developers need to do anything in this bucket?  Are permissions set appropriately on this bucket to allow (or disallow) that interaction?

Reply:
testers/developers don't need to do anything now, and the bucket is properly permissioned.  We are waiting on SIA approval and the final hosting projects to be created; once that's in place and deploy scripts are running,  I'll  go  back and update  all of this config.